src/genie/libs/parser/iosxe/tests/ShowRunningConfigWlan/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,47 @@
+expected_output = {
+    "profile_name": {
+        "lizzard_Fabric_F_90c76354": {
+            "wlan_id": 17,
+            "ssid": "lizzard",
+            "config": [
+                "radio dot11a",
+                "security ft",
+                "security wpa akm ft dot1x",
+                "security dot1x authentication-list dnac-cts-list",
+                "no shutdown",
+            ],
```

Comment:
this better to be key-value pair like other parsers.
I don't have the device where I can check these command. but something like below?
```
"radio": "dot11a",
"security": "ft",
"security_wpa_akm_ft": "dot1x",
"security_dot1x_authentication_list": "dnac-cts-list",
"shutdown": False, # if has `no`, set False as boolean
```
But need to check more based on command syntax. open to hear comments on this. any concerns with this way?

Reply:
I tried that at first, but like you said they depend on command syntax. This output comes from a C9800 Wireless Controller, but I don't have access to the device. Are we able to pull a list of available commands from Cisco documentation?

Reply:
@robschn unfortunately, as far as I know, no useful way to pull a list of commands.
I'm thinking idea to tackle this kind of show running-config parser. please standby. it might take a time, but will update.

Reply:
Sounds good! I appreciate the support! 😄

Reply:
Just update. still working on the idea.

Reply:
No problem! Still working through your edits so no rush

Reply:
sorry, still couldn't have a time to work on this. Let me know if you really need this parser. I will try to prioritize.

Reply:
We can wait on your solution for parsing run commands. I'm going to check internally if we have access to a virtual IOS that has this particular command.

Reply:
sorry, still pending. I hope I can create a time to work on this next week.

Reply:
thanks for the update.

Reply:
thanks for waiting and sorry for delaying. keep busy with other stuff... Let me know if you need to prioritize on this. or we might want to consider to merge this parser until my new way comes in.

Reply:
sorry for delayed update. I finally developed API `get_structure_output()` to parse config and automatically generate the structure data based on spacing. not perfect, but reduce the errort to develop parsers mainly for `show run xxx`. The API will be ready in 21.3 release (end of this month).

```
In [6]: out = dev.api.get_structure_output(command='show running-config wlan')
(snip)

In [7]: print(json.dumps(out, indent=4, sort_keys=True))
{
    "wlan": {
        "alfa_Global_F_953d5d2b": {
            "17": {
                "alfa": {
                    "radio": {
                        "dot11a": true
                    },
                    "security": {
                        "dot1x": {
                            "authentication-list": {
                                "dnac-cts-list": true
                            }
                        },
                        "ft": {
                            "over-the-ds": true
                        }
                    }
                }
            }
        },
        "internet_Fabric_F_522662da": {
            "19": {
                "internet": {
                    "mac-filtering": {
                        "default": true
                    },
                    "security": {
                        "ft": {
                            "adaptive": false
                        },
                        "wpa": {
                            "akm": {
                                "dot1x": false
                            },
                            "wpa2": {
                                "ciphers": {
                                    "aes": false
                                }
                            }
                        }
                    },
                    "shutdown": false
                }
            }
        },
        "lizzard_Fabric_F_90c76354": {
            "17": {
                "lizzard": {
                    "radio": {
                        "dot11a": true
                    },
                    "security": {
                        "dot1x": {
                            "authentication-list": {
                                "dnac-cts-list": true
                            }
                        },
                        "wpa": {
                            "akm": {
                                "ft": {
                                    "dot1x": true
                                }
                            }
                        }
                    },
                    "shutdown": false
                }
            }
        },
        "lizzard_l_Fabric_F_0e98c2cd": {
            "18": {
                "lizzard_2": {
                    "no": {
                        "broadcast-ssid": true
                    },
                    "radio": {
                        "dot11bg": true
                    },
                    "security": {
                        "dot1x": {
                            "authentication-list": {
                                "dnac-cts-list": true
                            }
                        }
                    },
                    "shutdown": false
                }
            }
        }
    }
}
```

Reply:
Sounds good! Thanks for all the hard work! We can close this PR now.